src/server/socketHandlers/scConnectionHandler.js

```diff
@@ -57,40 +51,44 @@ export default function scConnectionHandler(exchange, sharedDataLoader) {
       asyncIterator.return();
       delete socket.subs[opId];
     });
-    socket.on('disconnect', () => {
+    socket.on('disconnect', async () => {
```

Comment:
No reason to use `async` if there isn't an `await` inside the function, and if we're not returning a promise.

Reply:
well the linter definitely failed. nice catch!